Automattic/jetpack
# Scan notices: fix the location of the enqueued js files

## Proposed changes:

This fixes an issue introduced in #34072.

- The non-minified file (accessible when `SCRIPT_DEBUG` is set to true) should load from inside the modules directory.
- The minified file should load from _inc/build, located at the root of the Jetpack plugin.


### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

This is not necessarily easy to test. I would recommend the following:

* Get a site set up with a Jetpack Scan plan, and connected to WordPress.com.
* In `wp-config.php`, set `SCRIPT_DEBUG` to true.
* Edit `projects/plugins/jetpack/modules/scan/class-admin-bar-notice.php` and comment out these lines:
https://github.com/Automattic/jetpack/blob/63f39b656d31524e0bb63a0be5817457af0f0fc9/projects/plugins/jetpack/modules/scan/class-admin-bar-notice.php#L103-L105
* Open your browser's network dev tools, filter for js files, with a name including `admin-bar-notice`
* Load wp-admin.
    * You should see a file successfully enqueued: `wp-content/plugins/jetpack/modules/scan/admin-bar-notice.js?minify=false&ver=1`
* Now edit `wp-config.php` again, and set `SCRIPT_DEBUG` to false.
* Load wp-admin again.
    * You should now see another file successfully enqueued:
`wp-content/plugins/jetpack/_inc/build/scan/admin-bar-notice.min.js?minify=false&ver=1`
